From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Thu, 2 Jul 2009 18:08:56 +0200
Subject: [PATCH] make autoreconf copy files

so we don't end up with symlinks in the source
---
 autogen.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 731e44d..da004d6 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -20,10 +20,10 @@
 
 set -e
 set -v
-export AUTOMAKE='automake --foreign'
+export AUTOMAKE='automake --foreign --copy'
 mkdir -p daemon/m4
 aclocal
-libtoolize
+libtoolize --copy
 autoreconf -i
 pushd daemon
 autoreconf -i
-- 
